widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizer.kt

```diff
@@ -195,12 +195,16 @@ internal class ActivityWatcherForCallVisualizer(
         }
     }
 
-    override fun dismissAlertDialog(manualDismiss: Boolean) {
+    override fun dismissAlertDialog() {
         Logger.d(TAG, "Dismiss alert dialog")
         alertDialog?.dismiss()
         alertDialog = null
     }
 
+    override fun removeDialogFromStack() {
+        dialogController.dismissCurrentDialog()
+    }
```

Comment:
It is very confusing to me when which method should be called. Could you please rename methods or write comments in the code?

But just for me to understand:
`dismissAlertDialog` - just removes the current on-screen dialog if has any, correct?
`removeDialogFromStack` - will in the end call the `dismissAlertDialog` but through a proper workflow `DialogController.dismissCurrentDialog()` -> `DialogManager.dismissCurrent()` -> skiped few steps -> this classes `dismissAlertDialog`.

Reply:
dismissAlertDialog should not called from anywhere except onDialogControllerCallback.

For everyday use you would use removeDialogFromBackstack, because it will cause a dialog state change and call onDialogControllerCallback down the line